nuxeo-arender-rendition-apb/defaults/main.yaml

```diff
@@ -38,8 +38,8 @@ arender_jni_image_pullPolicy: IfNotPresent
 arender_pdfbox_image_pullPolicy: IfNotPresent
 
 arender_broker_resources: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1Gi"}}'
-arender_converter_resources: '{"limits": {"cpu": "1","memory": "6Gi"},"requests": {"cpu": "500m","memory": "4Gi"}}'
-arender_pdfbox_resources: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1Gi"}}'
+arender_converter_resources: '{"limits": {"cpu": "1.5","memory": "6Gi"},"requests": {"cpu": "500m","memory": "4Gi"}}'
+arender_pdfbox_resources: '{"limits": {"cpu": "1","memory": "1Gi"},"requests": {"cpu": "1","memory": "1.5Gi"}}'
```

Comment:
`requests` cannot be higher than `limits`

Reply:
ditto